package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "asset-compile": "ts-node tasks/asset-compile",
     "test": "mocha",
-    "fmt": "prettier --parser typescript --no-semi --print-width 100 --single-quote --write \"{src,plugins,tasks}/**/*.ts\"",
+    "fmt": "prettier --parser typescript --no-semi   --print-width 100 --single-quote --write \"{src,plugins,tasks}/**/*.ts\"",
```

Comment:
I don't think this diff does anything

Reply:
it did for the caching (yarn didn't run before so we had 32 byte cache entry, had to force it to get a new sha hash